app/src/main/res/layout/fragment_win.xml

```diff
@@ -42,11 +42,18 @@
             android:layout_height="wrap_content"
             android:text="@string/add_highscore_button" />
 
+        <Button
+            android:text="@string/share_score"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:id="@+id/shareScoreButton" />
```

Comment:
So far, we've used the snake-case (C style) namings in the XML files, like `share_score_button` and `win_score_text`.
The Android framework is smart enough to convert the object references into camel case (Java style) in the Kotlin classes:
`binding.winScoreText`

Reply:
Indeed. I have just updated it